### ISSUE TITLE
Initial Release

### ISSUE THREAD

**Issue by zkxs** [zkxs/runtime-frooxengine-mods#3]
This issue tracks the remaining work to be done before this repository is ready for its initial release.

- [x] Migrate everything into this monorepo, preserving commit history.
  - [x] make it work with `dotnet build` for eventual CI/CD support. This requires a weird csproj setup.
  - [x] make it work in VS/VSC. This requires a weird .sln setup.
  - [x] perform pending project renames. A lot of my mods have the word "Neos" as a prefix, which is very redundant information when sitting in a nml_mods folder.
- [x] Recreate NML's config API as a separate bepinex mod
- [ ] Remove NML's config API and make it depend on the config mod instead
- [ ] Rewrite NML to run on BepinEx vs the Neos Plugin hack
  - [ ] remove compatibility hash logic
  - [ ] intentionally load mods at the end of plugin init to avoid issues where NML mods are loaded too early
  - [ ] do *something* to the mod class hiding logic. Remove? Separate into separate mod? I'm not sure.
- [ ] Rewrite mods to run on BepinEx with the config mod instead of using NML. One test mod (NeosModTest) will be kept on NML for testing.
  - [ ] NoResetScale
  - [ ] ~~ExportNeosToJson~~ (superseded by [LocalStorage](https://github.com/art0007i/LocalStorage), not going to port)
  - [ ] SsaoDisable
  - [ ] TrackerIdStabilizer
  - [ ] LocomotionRename
  - [ ] MotionBlurDisable
  - [ ] PlatformSpoof
  - [ ] VoiceQuality
  - [ ] ContactsSort
  - [ ] CloudHomeAccessLevel
  - [ ] VoiceVolumeOverride
  - [ ] DesktopToolShortcutRemapper
  - [ ] NeosModConfigurationExample
  - [ ] ~~MyHostMyRules~~ (not going to port)
- [ ] Neos -> Resonite fixes
  - [ ] disable or fix splash changer from NML
  - [ ] validate all mods actually function as expected in Resonite
- [ ] make sure license & copyright info is properly preserved and conveyed
  - [x] each subproject has its own license
  - [ ] make sure license text is standard (e.g. don't use slightly different GPL copy/pastes in different projects, that's weird)
  - [ ] add copyright notices to the top of each source file, as per GPL recommendation
- [ ] Archive original mod project repos once this repo is ready
